local/src/test/scala/com/salesforce/op/local/OpWorkflowModelLocalTest.scala

```diff
@@ -153,7 +148,8 @@ class OpWorkflowModelLocalTest extends FlatSpec with PassengerSparkFixtureTest w
 
   private def assert(
     scores: Array[Map[String, Any]],
-    expectedScores: Array[(Prediction, RealNN, RealNN, Text)]
+    expectedScores: Array[(Prediction, RealNN, RealNN, Text)],
+    prediction: FeatureLike[Prediction]
```

Comment:
It doesn't look like you used this. Is it needed?

Reply:
it's used under `[prediction.name -> predV.value](https://github.com/salesforce/TransmogrifAI/blob/c4919cfb17824750622e33fe537e4d4f45463940/local/src/test/scala/com/salesforce/op/local/OpWorkflowModelLocalTest.scala#L158)`